src/app/model/StalemateHasOccurred.ts

```diff
@@ -0,0 +1,8 @@
+import { King } from './pieces';
+import { Square } from './Types';
+
+export type StalemateHasOccurred = {
+  eventType: 'StalemateHasOccurred';
+  king: King;
+  onSquare: Square | undefined;
```

Comment:
@lamparina czy Stalemate musi dotyczyc króla? To chyba oznacza, że nikt się nie może ruszyć, nie? Czyli żadne dane nie są potrzebne. Może ewentualnie strona, która nie może się ruszyć - jak ja to rozumiem.

Reply:
pat nie musi dotyczyć króla, wg definicji to jest sytuacja w której gracz nie może wykonać żadnego 'legalnego' ruchu, ale jego król nie jest szachowany

Reply:
done